setup.cfg

```diff
@@ -1,2 +1,8 @@
 [bdist_wheel]
 universal = 1
+
+[flake8]
+per-file-ignores =
```

Comment:
```suggestion
# E701 Multiple statements on one line (colon)
# E702 Multiple statements on one line (semicolon) 
per-file-ignores =
```

It would be nice to document what the codes mean.

Reply:
Done, but I added some more detailed reasoning as well.